wordle-web/src/components/LetterBase.vue

```diff
@@ -1,6 +1,16 @@
 <template>
-  <v-btn :color="props.color">
-    {{ props.char }}
+  <v-btn
+    style="
```

Comment:
Avoid inline styles, this is bad practice. If you need to use css, put it in a css class.